cmake/cached_globals.cmake

```diff
@@ -19,9 +19,18 @@ set(GLOB_CURRENT_YEAR ${GLOB_CURRENT_YEAR} CACHE STRING "Current year")
 
 # .NET config
 if(WIN32 AND BUILD_DOTNET_WRAPPER)
-  set(GLOB_DOTNET_TARGET_FRAMEWORK "net481" CACHE STRING ".NET target framework")
-  set(GLOB_DOTNET_TARGET_FRAMEWORK_VERSION "v4.8.1" CACHE STRING ".NET target framework version")
+  # .NET SDK
   set(GLOB_DOTNET_SDK "Microsoft.NET.Sdk" CACHE STRING ".NET SDK")
+  # .NET target frameworks
+  # Note: Even if the list consists of a single element, make sure to terminate with a semicolon
+  #       For ex: set(LIST_OF_DOTNET_TARGET_FRAMEWORKS "net481;" CACHE STRING "Supported .NET target framework")
+  #       Otherwise, Visual Studio will not create subdir net481 under Release (of whichever build config is used) in the CMake binary dir
+  #       <TargetFramework>net481</TargetFramework> works (notice the singular form in xml element "TargetFramework")
+  #       <TargetFrameworks>net481</TargetFrameworks> does not work (notice the plural form in xml element "TargetFrameworks")
+  #       <TargetFrameworks>net481;</TargetFrameworks> works (notice the plural form in xml element "TargetFrameworks" and the semicolon after "net481")
+  #       Is this a VS bug? "TargetFramework" should be a special case of "TargetFrameworks" when the length of the list is 1.
+  set(GLOB_LIB_DOTNET_TARGET_FRAMEWORKS "netstandard2.0;net481" CACHE STRING "Library .NET target framework")
+  set(GLOB_TEST_DOTNET_TARGET_FRAMEWORKS "net481;" CACHE STRING "Test .NET target framework")
+  # .NET language version
   set(GLOB_CSHARP_LANGUAGE_VERSION "7.3" CACHE STRING "C# Compiler Options for language feature rules") 
```

Comment:
Is it possible to set language version per project? For the dotnetstandard we could set it to the latest and for the net481 we would need to set it to 7.3 (that is the latest supported for net481)

Reply:
This is actually unused.

it can be set for an entire solution via 
```
target_compile_options(<target> PRIVATE "/langversion:${GLOB_CSHARP_LANGUAGE_VERSION }")
```
Setting it per target framework can be pron done for example via:
```
<PropertyGroup Condition="'$(TargetFramework)' == 'net48'">
    <LangVersion>7.3</LangVersion>
</PropertyGroup>
```
But CMake does not seem to be able to set conditional properties. 

According to [this article](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/language-versioning)
```
The latest C# compiler determines a default language version based on your project's target framework or   
frameworks. Visual Studio doesn't provide a UI to change the value, but you can change it by editing the csproj file.
```
But again, I do not know if this is done per platform. It makes sense to have the highest possible supported by all target platforms.